api/models/index.js

```diff
@@ -11,6 +11,7 @@ export const init = () => {
   global.Promise = Promise
 
   global.bookshelf = Bookshelf(Knex(knexfile[process.env.NODE_ENV]))
+  global.bookshelf.plugin('bookshelf-returning')
 
```

Comment:
@levity just wanted to flag this for your attention, in case it has side-effects with other things we're doing that I don't know about! I just wanted to avoid fetch-after-create and fetch-after-update hits on the db, by using `{ returning: '*' }`. Potentially we could save quite a few hits from other models too by doing this.